internal/output/prometheusrw/remote/client_test.go

```diff
@@ -0,0 +1,229 @@
+package remote
+
+import (
+	"context"
+	"io"
+	"math"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+	"time"
+
+	"go.k6.io/k6/internal/output/prometheusrw/stale"
+
+	prompb "buf.build/gen/go/prometheus/prometheus/protocolbuffers/go"
+	"github.com/golang/snappy"
```

Comment:
This likely should be the klaus snappy instead - which is already used through the project. This PR now adds this as a dependancy while it didn't before. Specifically because of this test.

Reply:
Good catch :handshake: